woocommerce/woocommerce
# Set US:CA as the default store location

### All Submissions:






### Changes proposed in this Pull Request:



We want to default store location to US California. This is mainly because WC Pay uses this locale to determine what to advertise.

Closes #29499

### How to test the changes in this Pull Request:

1. You can probably do this locally by deleting some options but I find it easier just to test it on Jurassic Ninja. So set one up.
2. Pull down this PR and run `npm run build:zip` to build a zip package to install on your JN site.
3. After activating the plugin, you will see the Setup Wizard.
4. Click the link at the bottom to skip it.
5. Go to `/wp-admin/admin.php?page=wc-settings`
6. Ensure the Country / State shows `United States (US) - California` as the default.

### Other information:




### Changelog entry

> Tweak - Default store location to US California.
